CMakeLists.txt

```diff
@@ -154,10 +157,12 @@ endif(NOT WIN32)
 # permissive- : Std compliant parsing, warnings (W3) set by CMake defaults
 add_compile_options($<$<CXX_COMPILER_ID:MSVC>:-permissive->)
 
-option(PRECOMPILED_HEADERS "Set whether to build with precompiled headers" ON)
+option(PRECOMPILED_HEADERS "Set whether to build with precompiled headers" OFF)
```

Comment:
Why does this need to be off?

Reply:
I don't know about other systems, but for me it kind of breaks `ccache`. It always rebuilds, so I though OFF is a better default.

Reply:
Ok, thanks for explaining. I had thought the changes were relevant to the dynamic library changes.

Reply:
Maybe just add a comment that this is specific to ccache?

Reply:
This has been done in a follow-up branch, which is in `main` already.